### ISSUE TITLE
Add support for RPM sentence

### ISSUE THREAD

**Issue by tkurki** [SignalK/signalk-to-nmea0183#104]
https://www.tronico.fi/OH6NT/docs/NMEA0183.pdf

#### RPM Revolutions

```
       1 2 3   4   5 6
       | | |   |   | |
$--RPM,a,x,x.x,x.x,A*hh
1) Source; S = Shaft, E = Engine
2) Engine or shaft number
3) Speed, Revolutions per minute
4) Propeller pitch, % of maximum, "-" means astern
5) Status, A means data is valid
6) Checksum
```

**Comment by Shelis1976** [comment]
I'm little confused. When I use emulator and the data comes from nmea183 and it is passing thru SignalK to the TCP connections - all is ok on the QTVLM - RPM works. If I connect my Engine HAt who produce only SignalK sentence to Signal K serwer RPM on QTVLM - no works. What is wrong? I think SignalK serwer not decode from Signal K to nmea183 ... ???? 
Mayby there is some problem with SignalPaths. When I have path tanks.fuel.A.currentLevel - Open CPN dont recognise them. But When I put tanks.fuel.1.currentLevel all is Ok.  I mean engine dashboard not DasboardSK - In this we can choose paths

**Comment by tkurki** [comment]
This issue just marks that ”somebody should implement RPM” it is not supported yet.

**Comment by msallin** [comment]
@ManfredRad still interested in contributing? Im glad to work with you to get the PRs in shape. And I need a beta tester then, since Im not able to test it myself (beside unit tests ofc).

**Comment by ManfredRad** [comment]
@msallin yes I am interested in contributing. The code has been tested by me already. I will check again later this week (currently not at home) to verify that all is okay.
